js/components/QuestionTypes/DateQuestionAndroid.js

```diff
@@ -0,0 +1,87 @@
+
+import React, {
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+  DatePickerAndroid,
+  TouchableWithoutFeedback,
+} from 'react-native'
+import Styles from '../../styles/Styles'
+import Button from 'apsl-react-native-button'
+import moment from 'moment'
+
+const DateQuestionAndroid = React.createClass ({
+  propTypes: {
+    id: React.PropTypes.string.isRequired,
+    text: React.PropTypes.string.isRequired,
+    value: React.PropTypes.object,
```

Comment:
will get a yellowbox warning when the value is a string, as in DatetimeQuestionAndroid.js

```    
value: React.PropTypes.oneOfType([
      React.PropTypes.string,
      React.PropTypes.instanceOf(Date)
    ]),
```